coalib/testing/LocalBearTestHelper.py

```diff
@@ -33,12 +33,28 @@ def get_results(local_bear,
                 create_tempfile=True,
                 tempfile_kwargs={},
                 settings={}):
+    deps_results = dict()
```

Comment:
you can move that below into the `if`, as in the `else` clause you set it to `None` anyway^^